extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -324,6 +321,13 @@ export class AmpAdExit extends AMP.BaseElement {
    * @private
    */
   init3pResponseListener_() {
+    if (getMode().runtime == 'inabox') {
```

Comment:
can you add a TODO to remove this when https://github.com/ampproject/amphtml/issues/11436 is resolved?

Reply:
Happy to, though please clarify: #11436 is about amp-analytics living inside amp-ad. Will fixing that one fix this issue, which is about amp-ad-exit living inside amp-ad?

Reply:
I thought so. Is there any other reason that this check exists? In other words, does amp-ad-exit only care whether analytics are available, or does it really care whether it's inside an amp-ad vs. inabox context?

Reply:
I will have to defer to @zhucl for that answer. From an email earlier today, "As amp-ad-exit is also used in the amp-in-a-box case, can we remove that assert?".

Reply:
From @zhucl's email this morning, it looks like this "if" should not go away.

Reply:
I still don't understand why 3P analytics can only be used inside an amp-ad and not inabox. Since non-AMP pubs represent far more traffic it would be good to have this usable in that context.

Reply:
Ok, I see what you mean now, and I agree. I will add the TODO and I envision in the future, we just use the resource ID of *something* that encloses amp-ad-exit, which will not need to be an amp-ad.

Reply:
Cool, thanks